aether-ui/aether/ui/assets/apps/pipeline/sections/Settings.jsx

```diff
@@ -230,17 +210,24 @@ class Settings extends Component {
             }
           </div>
 
-          <button
```

Comment:
In this page, each section is a form, so this "save" button refers only to the "name" field, could you move it to its section? :grimacing:

Reply:
but this comes with the constraint, that the save action needs to be placed right after the text input. and then we need another button at the bottom. 
It changes the flow and adds noise, that is why I am not totally happy about it.
But it is also not a show-stopper, so I will bring it back and maybe do something with sizes and wording.

Reply:
We don't need another save button at the bottom because there's nothing on the page to save apart from the title. If you press _Create identity contract_, the relevant stuff is created in UI without needing to press another _save_ button.

In fact, since the _Save_ button is only saving the name, could we think about taking it out and just saving whenever the name changes? Or maybe do something like Jira's individual button for the title field?

![image](https://user-images.githubusercontent.com/1815204/53745217-dfae2e80-3e9e-11e9-82df-fb2fc5e84df3.png)

(Can't believe I'm taking UX ideas from Jira...)

Reply:
Yes, actually that was something I was thinking of, just having a little check button at the end of the textfield.
And at the bottom I’ll put a big „finish“ button, which actually does nothing but closing the page.